docs/xtrabackup-versioning-scheme.md

```diff
@@ -1,4 +1,4 @@
-### Percona XtraBackup 2.4 versioning scheme:
+# Percona XtraBackup 2.4 versioning scheme:
 
 Here is what the numbers in the product version specification mean:
```

Comment:
See the PXC version. This sentence is out of place and shouldn't end with a colon.

Reply:
Updated